src/docker/composeappengine.cc

```diff
@@ -51,11 +51,11 @@ bool ComposeAppEngine::isRunning(const App& app) const {
       LOG_WARNING << "App: " << app.name << ", no services in docker file!";
       return false;
     }
-    DockerClient client(app.name);
+
```

Comment:
now that the docker-client doesn't get instantiated each call, its `root_` will be static. This looks to be a nice performance boost. however - don't we need some kind of `docker_client_->refresh()` type method so it picks up changes in the system?